testdata/kubernetes/BUILD

```diff
@@ -0,0 +1,5 @@
+filegroup(
+    name="kubeconfig",
+    srcs=["config"],
```

Comment:
where does the `src` come from? It seems there should be a file somewhere called config?